### PR TITLE
Feature/develop/task2

### PR DESCRIPTION
**Pull request by Bozhyk-Oleh**
lab2

### CODE REVIEW

File: Client/Client.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (andrii-s-shpitser-lpnu-ua) [inline Client/Client.sln:1]: please add CI scripts to build your project

File: master_server/master_server.ino [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+byte responseNOB, SLAVE_ID = 8; ;
+
+void setup() {
+  wdt_disable;
+  Wire.begin();        // join i2c bus (address optional for master)
+  pinMode(LED_BUILTIN, OUTPUT);  
+  wdt_enable (WDTO_2S);
+}
+
+void loop() {
```
Comment (andrii-s-shpitser-lpnu-ua) [inline master_server/master_server.ino:42]: loop is too big, please split it into smaller functions.

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

File: .github/workflows/build_task2.yml [hunk 3]
```diff
@@ -5,7 +5,7 @@ on: [workflow_dispatch]
 jobs:
   build:
 
-    runs-on: microsoft-latest
+    runs-on: ubuntu-latest
```
Comment (andrii-s-shpitser-lpnu-ua) [inline .github/workflows/build_task2.yml]: will it generate EXE for Windows

Comment (Bozhyk-Oleh) [reply .github/workflows/build_task2.yml]: -- no  i get  an error when trying to build the project  in ci 
  C:\Program Files\dotnet\sdk\6.0.402\Microsoft.Common.CurrentVersion.targets(1220,5): error MSB3644: The reference assemblies for .NETFramework,Version=v3.0 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks

Comment (Bozhyk-Oleh) [reply .github/workflows/build_task2.yml]: i did try something new and now it is generating a exe file, please check it 
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3272637944/jobs/5383887733

Comment (andrii-s-shpitser-lpnu-ua) [reply .github/workflows/build_task2.yml]: Good, but how I can download EXE?

Comment (Bozhyk-Oleh) [reply .github/workflows/build_task2.yml]: DONE!
https://github.com/Bozhyk-Oleh/csdt2223ki46bozhyk02/actions/runs/3276904833

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**